front/app/containers/Admin/projects/components/PDFExportModal.tsx

```diff
@@ -0,0 +1,109 @@
+import React, { useState } from 'react';
+
+// components
+import Modal from 'components/UI/Modal';
+import { Box, Button, Text } from '@citizenlab/cl2-component-library';
+
+// i18n
+import { FormattedMessage } from 'utils/cl-intl';
+import messages from './messages';
+
+// form
+import Checkbox from 'components/HookForm/Checkbox';
+import { useForm, FormProvider } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { object, boolean } from 'yup';
+import { isCLErrorsIsh, handleCLErrorsIsh } from 'utils/errorUtils';
+
+const DEFAULT_VALUES = {
+  name: false,
+  email: false,
+} as const;
+
+interface Props {
+  open: boolean;
+  onClose: () => void;
+  onExport: (params: { name: boolean; email: boolean }) => Promise<void>;
+}
+
+const PDFExportModal = ({ open, onClose, onExport }: Props) => {
+  const [loading, setLoading] = useState(false);
+
+  const schema = object({
+    termsAndConditionsAccepted: boolean(),
+    privacyPolicyAccepted: boolean(),
+  });
+
+  const methods = useForm({
+    mode: 'onSubmit',
+    defaultValues: DEFAULT_VALUES,
+    resolver: yupResolver(schema),
+  });
+
+  const handleExport = async ({
+    name,
+    email,
+  }: {
+    name: boolean;
+    email: boolean;
+  }) => {
+    setLoading(true);
+
+    try {
+      await onExport({ name, email });
+      setLoading(false);
+      onClose();
+    } catch (e) {
+      setLoading(false);
+
+      if (isCLErrorsIsh(e)) {
+        handleCLErrorsIsh(e, methods.setError);
+        return;
+      }
+    }
+  };
+
+  return (
+    <Modal fullScreen={false} width="580px" opened={open} close={onClose}>
+      <FormProvider {...methods}>
+        <form onSubmit={methods.handleSubmit(handleExport)}>
+          <Box p="0px" w="100%">
+            <Text mb="20px" mt="0px" w="500px">
```

Comment:
We still need to decide what we put in here exactly for the idea form and the survey. Just put some placeholder text in here for now